core/src/main/java/org/mule/runtime/core/policy/DefaultPolicyStateHandler.java

```diff
@@ -0,0 +1,42 @@
+/*
+ * Copyright (c) MuleSoft, Inc.  All rights reserved.  http://www.mulesoft.com
+ * The software in this package is published under the terms of the CPAL v1.0
+ * license, a copy of which has been included with this distribution in the
+ * LICENSE.txt file.
+ */
+package org.mule.runtime.core.policy;
```

Comment:
Yes, we should start doing it. Lets discuss it next scrum. I can do that refactor in the next commit related to policies.